projects/plugins/jetpack/scss/atoms/_buttons.scss

```diff
@@ -7,7 +7,7 @@
 	transition: all .1s ease-in-out;
 }
 
-.jp-button {
+%jp-button {
```

Comment:
This will have the same problem as the boost one above.

Reply:
e0f7bf441a